README.md

```diff
@@ -44,35 +52,114 @@ Install-PSResource SecretManagement.1Password
 ```
+Next are the detials provided in the registration:
+* **Name**: Name of the SecretManagement vault. This will be the name to use when managing secrets from the SecretManagement powershell cmdlets.
+* **ModuleName**: Name of the PowerShell extension module that will be interacting with the underlyging secrets source. In this case, as the source will be "1Password" the extension module name must be "SecretManagement.1Password".
+* **VaultParameters**: Optional. Details required by the extension module to access the source secrets. See the section [Vault parameters](#Vault-parameters) for details specific for the SecretManagement.1Password extension module (used to access 1Password).
+
+**Note**: The name given to the SecretManagement vault (provided with the `Name` parameter) doesn't need to match the name of an existing vault in 1Password. Considering that the SecretManagement module supports multiple sources, it may be useful to prefix each of its vaults with a word that allows to know the source. For instance, in the case of 1Password vaults, the SecretManagement vaults can be named as "1Password: VaultName".
 
-**Note**: The name you provide the `Name` parameter needs to match an existing vault in 1Password.
-If you want to access more than oen 1Password vault you need to register them separately with `Register-SecretVault`
+It is recommended to regiser one SecretManagement vault for each 1Password vault that need to be accessed.
 
 
 ### Vault parameters
 
 The module also has the following vault parameter that must be provided at registration.
 
+```pwsh
+$vaultParameters = @{
+    AccountName = 'myaccount.1password.com'
+    OPVault = 'MyVaultName'
+}
+```
+
 #### AccountName
 
-Your 1Password account name.
+Optional. Specifies what 1Password account to connect to when accessing secrets. It is common to have a corporate and a personal account. This parameter allows to select one of your accounts. If this parameter is not provided, then the default 1Password account will be used.
+
+The 1Password account name can be found in the URL used to access 1Password as follows:
 
 ```
 https://myaccountname.1password.com/
-        ^^^^^^^^^^^^^
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 ```
+Corporate accounts are typically accessed through a URL like `https://myaccountname.1password.com/`. In this example, the account name is `myaccountname.1password.com`.
+
+Personal accounts typically have `my.1password.com` as account name.
+
+#### OPVault
+
+Name or Id of the 1Password vault associated with the SecretManagement vault.
+
+If this parameter is missing then the 1Password CLI will search on all vaults in the target account. 
+
+WARNING: Not linking the SecretManagement vault with a unique 1Password vault may cause issues because there may be more than one secret, stored in different 1Password vaults, sharing the same name. In that case, retrieval and updates uperations will have issues.
```

Comment:
```suggestion
> [!WARNING]
> Not linking the SecretManagement vault with a unique 1Password vault may cause issues because there may be more than one secret, stored in different 1Password vaults, sharing the same name. In that case, retrieval and updates uperations will have issues.
```
Probably a good opportunity to use the GitHub Flavored syntax for alerts.